<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regist</class>
 <widget class="QDialog" name="regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #f9f0fe;
}</string>
  </property>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwd2Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 2px;
	border: 1px, solid #000000;
}
QLineEdit:hover {
	border: 1px solid #34495E;
}</string>
   </property>
   <property name="placeholderText">
    <string>Your username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 2px;
	border: 1px, solid #000000;
}
QLineEdit:hover {
	border: 1px solid #34495E;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Your password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd2Edit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 2px;
	border: 1px, solid #000000;
}
QLineEdit:hover {
	border: 1px solid #34495E;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm your password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>Display Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #87CEFA;
	border-radius: 10px; 
	color: rgb(0, 0, 0); 
	font-family: &quot;Microsoft YaHei&quot; 
} 
QPushButton:hover { 
	background-color: #00BFFF; 
}</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
	background-color: rgb(255, 159, 198); 
	border-radius: 10px; color: rgb(0, 0, 0); 
	font-family: &quot;Microsoft YaHei&quot; } 
QPushButton:hover { 
	background-color: rgb(255, 133, 166); 
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="extLabel_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 2px;
	border: 1px, solid #000000;
}
QLineEdit:hover {
	border: 1px solid #34495E;
}</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="extLabel_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
   </property>
   <property name="text">
    <string>IPAddress</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 2px;
	border: 1px, solid #000000;
}
QLineEdit:hover {
	border: 1px solid #34495E;
}</string>
   </property>
   <property name="text">
    <string>6666</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>regist</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
